pkg/messaging/v1beta1/client.go

```diff
@@ -53,3 +54,10 @@ func (c *messagingClient) ChannelsClient() KnChannelsClient {
 func updateMessagingGVK(obj runtime.Object) error {
 	return util.UpdateGroupVersionKindWithScheme(obj, messagingv1beta1.SchemeGroupVersion, scheme.Scheme)
 }
+
+// BuiltInChannelGVKs returns the GVKs for built in channel
+func BuiltInChannelGVKs() []schema.GroupVersionKind {
+	return []schema.GroupVersionKind{
+		messagingv1beta1.SchemeGroupVersion.WithKind("InMemoryChannel"),
```

Comment:
According to the CRDs defined in knative messaging, `Channel` is another channel type ( although it has no specific implementation ) which has the label `messaging.knative.dev/subscribable: "true"` too. Here you don't regard it as a kind of build-in `Channel` type. I wonder if it will bring any inconsistency when `listBuiltInChannelTypes` is invoked.